packages/create-pantheon-decoupled-kit/src/templates/gatsby-wp/package.json.hbs

```diff
@@ -0,0 +1,61 @@
+{
+	"name": "{{kebabCase appName}}",
+	"version": "0.1.0",
+	"private": true,
+	"license": "GPL-3.0-or-later",
+	"description": "",
+	"homepage": "",
+	"bugs": "",
+	"author": "",
+	"repository": "",
+	"scripts": {
+		"build": "gatsby build",
+		"develop": "ENABLE_GATSBY_REFRESH_ENDPOINT=true gatsby develop -H 0.0.0.0",
+		"format": "prettier --write \"**/*.{js,jsx,ts,tsx,json,md}\"",
+		"start": "npm run develop",
+		"serve": "gatsby serve -p 8000",
+		"clean": "gatsby clean",
+		"prettier": "prettier \"**/*.{js,jsx,md}\" --check --ignore-path .prettierignore",
+		"prettier:fix": "prettier \"**/*.{js,jsx,,md}\" --write --ignore-path .prettierignore",
+		"test": "vitest run",
+		"test:watch": "vitest",
+		"update-snapshots": "vitest run --update --silent",
+		"coverage": "vitest run --coverage"
+	},
+	"dependencies": {
+		"gatsby": "^4.24.0",
+		"gatsby-plugin-image": "^2.24.0",
+		"gatsby-plugin-manifest": "^4.24.0",
+		"gatsby-plugin-offline": "^5.24.0",
+		"gatsby-plugin-pnpm": "^1.2.10",
```

Comment:
It would be great if we could only add this dependency if the package manager is `pnpm`. We already have the `which-pm-runs` dependency added. You should be able to use that dependency in the gatsby-wp generator to determine if this package should be added or not.